community/modules/compute/schedmd-slurm-gcp-v5-partition/main.tf

```diff
@@ -15,13 +15,17 @@
  */
 
 locals {
+  ghpc_startup_script = var.startup_script == "" ? [] : [{
```

Comment:
This type of logic previously caused a regression in the controller and login node (see #777).

When using startup script module, the script is not known at apply time so this variable is also not known at apply time. Parts of slurm depend on the filename being known at apply time.

Reply:
Updated